src/akamai/lib/AkamaiHDPlayer.as

```diff
@@ -246,12 +246,15 @@ package
       var caption : Object = info["parameters"];
       for (var key:String in caption)
       {
-        var captionText : String = caption[key];
-        if (captionText != null && _mode !="disabled")
+        if(key.substr(0,4)=="data")
```

Comment:
Before this fix, we were able to read captions from BYU TV which was coming in the format {data, language} since it was looking for the first string but we found out that CC's from Blizzard come in the format {endtime, data, wrapOption[true]} which meant we had to handle it by looking for "data". V3 player already handles it in this way but it was missed while implementing this on V4 player.